web/database/migrations/2018_05_10_051439_add_job_type_column_to_job.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddJobTypeColumnToJob extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('jobs', function (Blueprint $table) {
+            $table->integer('duration')->after('summary');
+            $table->dateTime('start_date')->after('duration')->nullable();
+            $table->dateTime('end_date')->after('start_date')->nullable();
+            $table->unsignedInteger('job_type_id')->nullable()->after('duration');
+            $table->foreign('job_type_id')->references('id')->on('job_types');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('jobs', function (Blueprint $table) {
```

Comment:
Drop the table too

Reply:
This is not migration for creating this table, just to add a few columns, will drop the columns instead.